hail/python/hail/experimental/vcf_combiner/vcf_combiner.py

```diff
@@ -459,6 +459,7 @@ def run_combiner(sample_paths: List[str],
                  target_records: int = CombinerConfig.default_target_records,
                  overwrite: bool = False,
                  reference_genome: str = 'default',
+                 contig_recoding: Dict[str, str] = None,
```

Comment:
Optional[Dict[Str, str]]

Reply:
Ah, thanks.  Optional use to be optional (heh) if the default initializer is None, but that's no longer recommended:

 > A past version of this PEP allowed type checkers to assume an optional type when the default value is None, as in this code:
 > `def handle_employee(e: Employee = None): ...`
 > This would have been treated as equivalent to:
 > `def handle_employee(e: Optional[Employee] = None) -> None: ...`
 > This is no longer the recommended behavior. Type checkers should move towards requiring the optional type to be made explicit.

https://www.python.org/dev/peps/pep-0484/#union-types